workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -246,7 +263,10 @@ def _get_prephysics_stepper(
                         PureMLStepper(model, self._timestep, hydrostatic)
```

Comment:
You mentioned in your comment that 

> you don't know the overriding variables it provides until model.predict is called, so I can't put this check in the Loop init

I didn't realize that the MultimodelAdapter/RenamingAdapters don't subclass fv3fit.Predictor and so don't have output variables as attributes. Seems like they should, but outside of this PR scope, I'll make an issue.